src/main/resources/META-INF/maven/site.vm

```diff
@@ -161,82 +166,86 @@
             </ul>
 #*      *##end
 ##
-#*      *##if ($topBarEnabled)
-            </div>
+#*      *##if (!$sideBarEnabled)
```

Comment:
Stupid question: Was has the `$topBarEnabled` been exchanged for `!$sideBarEnabled`?

Reply:
That was a bug I found - the open tag appears if !$sideBarEnabled, but the close tag didn't have the same conditional logic.

There is nothing stopping you having both true, so I polished that out.